src/Microsoft.TestPlatform.Common/Interfaces/Engine/ClientProtocol/IParallelProxyDiscoveryManager.cs

```diff
@@ -25,4 +29,11 @@ public interface IParallelProxyDiscoveryManager : IParallelOperationManager, IPr
             IEnumerable<TestCase> lastChunk,
             bool isAborted);
     }
+
+    public enum DiscoveryStatus
```

Comment:
Could we make this internal ? If not, please add documentation for the enum and for each of its values (I know the names are self descriptive but some additional context would be nice).

Reply:
I use it in different project, so should be public. Added more comments.